scripts/zephyr_module.py

```diff
@@ -299,6 +299,7 @@ def git_revision(path):
     meta['workspace'] = {}
     workspace_dirty |= zephyr_dirty
 
+    west_projects = west_projects()
```

Comment:
this will cause a reload of the manifest, even if we just did a load of the manifest for fetching Zephyr modules.

That seems unnecessary waste of resources.

Reply:
I think it makes sense to allow all combinations, knowing what takes precedence, which is pretty obvious from the code